### DISCUSSION TITLE
Windows 11 works!

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#571] [Show and tell]
Well, it does fail with an error. But, if 1 gigabyte of RAM is allocated to the VM, it will boot, show the Windows 11 logo for a few minutes, and go to the `32-bit CPU` error UI!
@copy

**Comment by raspiduino** [comment]
If you can't actually use it, it doesn't work.

Theoretically, you can run a 32 bit Linux on v86, then run `qemu-system-x86_64` on it to emulate Windows 11. But even QEMU without hardware acceleration running natively on a Linux host is still super slow when emulating Windows 11.

So If you set up that VM today, maybe a year later Windows 11 will boot :)

**Reply by hello-smile6** [reply]
@raspiduino It's decently slow _with_ hardware acceleration!

**Reply by leap0x7b** [reply]
lmao
